src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySaved/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const universityOfMelbourne: string = '../../../../ignitus-Assets/ignitus-Images/img-Png/universityOfMelbourne.png';
+export const universityOfPortsmouth: string = '../../../../ignitus-Assets/ignitus-Images/img-Png/universityOfPortsmouth.png';
```

Comment:
As mentioned why can't you have data structure like this.

`const opportunity = [{ 'title': '', 'description:'', 'src:'', subTitle: ''}, {} ,{} ...];`

what is the point having 4 different constants??